src/pilot/templates/DellNfv/hugepages.pp

```diff
@@ -0,0 +1,40 @@
+#class: dellnfv::hugepages
```

Comment:
Should add the standard licensing header to the top of this file.

Reply:
Added.